docs/docs/manual/facets.md

```diff
@@ -0,0 +1,284 @@
+
+For example, you may wish to create a numeric facet that rounds your value to the nearest integer, enter
+
+```round(value)```
+
+If you have two columns of numbers and for each row you wish to create a numeric facet only on the larger of the two, enter
+
+```max(cells["Column1"].value, cells[“Column2”].value)```
+
+If the numeric values in a column are drawn from a power law distribution, then it's better to group them by their logs:
+
+```value.log()```
+
+If the values are periodic you could take the modulus by the period to understand if there's a pattern:
+
+```mod(value, 7)```
+
+You can learn more about numeric-modification functions on the [Expressions page](expressions). 
+
+## Customized facets
+
+Customized facets have been added to expand the number of default facets users can apply with a single click. They represent some common and useful functions you shouldn’t have to work out using an [expression](expressions).
+
+All facets that display in the “Facet/Filter” sidebar can be edited by clicking on the “change” button to the right of the column title. This brings up the expressions window that will allow you to modify and preview the expression being used. 
+
+### Word facet
+
+Word facet is a simple version of a text facet: it splits up the content of the cells based on spaces, and outputs each character string as a facet:
+
+```value.split(" ")```
+
+This can be useful for exploring the language used in a corpus, looking for common first and last names or titles, or seeing what’s in multi-valued cells you don’t wish to split up. 
+
+Word facet is case-sensitive and only splits by spaces, not by line breaks or other natural divisions. 
+
+### Duplicates facet
+
+A duplicates facet will return only rows that have non-unique values in the column you’ve selected. It will create a facet of “true” and “false” values - true being cells that are not unique, and “false” being cells that are. The actual expression being used is
+
+```facetCount(value, 'value', '[Column]') > 1```
+
+Duplicates facets are case-sensitive and you may wish to filter out things like leading and trailing whitespace or other hard-to-see issues. You can modify the facet expression, for example, with:
+
+```facetCount(trim(toLowercase(value)), 'trim(toLowercase(value))', 'cityLabel') > 1```
+
+### Numeric log facet 
+
+Logarithmic scales reduce wide-ranging quantities to tiny scopes. A log transformation can be used to make highly skewed distributions less skewed. If your numerical data is unevenly distributed (say, lots of values in one range, and then a long tail extending off into different magnitudes), a numeric log facet can represent that range better than a simple numeric facet. It will break these values down into more navigable segments than the buckets of a numeric facet. This facet can make patterns in your data more visible.
```

Comment:
I'm not sure "Logarithmic scales reduce wide-ranging quantities to tiny scopes" is very clear. What are "tiny scopes"?

I think the description from wikipedia is better:
"A logarithmic scale (or log scale) is a way of displaying numerical data over a very wide range of values in a compact way"